src/main/java/seedu/address/model/assignment/Schedule.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.model.assignment;
+
+public class Schedule {
+    private String dummyDeadline = "01-01-2000 0101";
+    private boolean schedule;
+    private Deadline suggestedStartTime;
+    private Deadline suggestedEndTime;
+
+    /**
+     * Constructor
+     * Returns a false schedule with dummy value
+     */
+    public Schedule() {
+        schedule = false;
+        suggestedStartTime = new Deadline(dummyDeadline);
+        suggestedEndTime = new Deadline(dummyDeadline);
+    }
```

Comment:
Is this constructor used for testing?

Reply:
Yes I was using it for testing, have remove the dummy test.